paasta_tools/cli/cmds/info.py

```diff
@@ -94,14 +94,9 @@ def get_deployments_strings(service: str, soa_dir: str) -> List[str]:
         )
         service_mode = service_config.get_mode()
         for cluster in deployments_to_clusters(deployments):
-            if service_mode == "tcp":
-                service_port = service_config.get("proxy_port")
+            if service_mode == "http" or service_mode == "https":
```

Comment:
```suggestion
            if service_mode in {"http", "https"}:
```
or 

```
TESTABLE_SERVICE_MODES = {"http", "https"}
...
            if service_mode in TESTABLE_SERVICE_MODES:
```

would be a little less repetitive, but this is fine as-is too :)